packages/react-notifications/src/components/InAppMessaging/BannerMessage/__tests__/BannerMessage.test.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import * as UiReactModule from '@aws-amplify/ui-react';
+
+import { useMessageProps } from '../../hooks';
+import { BLOCK_CLASS } from '../constants';
+import { BannerMessage } from '../BannerMessage';
+import { BannerMessageProps } from '../types';
+
+jest.mock('../../hooks/');
+jest.mock('../../MessageLayout', () => ({
+  MessageLayout: () => 'MessageLayout',
+}));
+
+type Alignment = BannerMessageProps['alignment'];
+type Position = BannerMessageProps['position'];
+type Locations = [Alignment, Position][];
+
+const BANNER_ALIGNMENTS: Alignment[] = ['left', 'center', 'right'];
+const BANNER_POSITIONS: Position[] = ['top', 'middle', 'bottom'];
+
+const BANNER_LOCATIONS = BANNER_ALIGNMENTS.reduce(
+  (acc: Locations, alignment) => {
+    BANNER_POSITIONS.forEach((position) => acc.push([alignment, position]));
+    return acc;
+  },
+  []
+);
+
+const TEST_PROPS: BannerMessageProps = { layout: 'TOP_BANNER' };
+
+const mockUseMessageProps = useMessageProps as jest.Mock;
+
+const useBreakpointValueSpy = jest.spyOn(UiReactModule, 'useBreakpointValue');
```

Comment:
Earlier, when we had InApp code in `ui-react`, we just imported `useBreakpointValue` from a relative path and mocked it. Changed it to now import it from `ui-react` and set a spy instead.

Same for other tests in `ui-react-notifications`.